### PR TITLE
Send Email Reminders to users

### PR DESCRIPTION
**Pull request by kofiarkoh**
This feature introduces  the feature of sending email reminders to users based on upcoming fitness events for the day using celery beats

### CODE REVIEW

**Comment by lingjunliu** [comment]
umm you're trying to merge this branch to "main" branch...
I will close this pull request

**Comment by kofiarkoh** [comment]
no need to close, i can just change the base branch.

**Review by habib-itopa — APPROVED** [review]
Looks good to me!

**Comment by lingjunliu** [comment]
Does it send the email reminder before an event starts? Or exactly the starting time of an event?

File: .gitignore [hunk 1]
```diff
@@ -3,4 +3,5 @@ package-lock.json
 __pycache__
 .idea
 .DS_Store
-.env
\ No newline at end of file
+.env
+celerybeat-schedule.db
```
Comment (lingjunliu) [inline .gitignore:7]: I noticed this file added in ".gitignore", but you didn't add in the commit.
What is the file for? Is this file automatically generated? or we need to add this to our repository as well?

Comment (kofiarkoh) [reply .gitignore:7]: this is an automatically generated db file by celery which we dont need to track. see my previous [commit](https://github.com/CSC510-GROUP-40/FitnessApp/pull/4/commits/2bf55c51805437be7a58cab7dd8f53ef52cf877e)

**Comment by kofiarkoh** [comment]
> Does it send the email reminder before an event starts? Or exactly the starting time of an event?

so this is just a reminder so its supposed to send the reminder once on the day  of the event before it starts. 

but for testing purpose on you local machine, i've set it to run every 5 seconds instead of daily.... we can later move this to and enviroment confirguation

**Review by lingjunliu — APPROVED** [review]

**Pull request merged**